tests/generation/test_utils.py

```diff
@@ -3348,11 +3348,13 @@ def test_default_max_length_warning(self):
 
         # Explicitly setting max_length to 20 -> no warning
         with warnings.catch_warnings(record=True) as warning_list:
+            warnings.simplefilter("ignore", category=FutureWarning)
```

Comment:
Instead of ignoring the warning, let's make sure `generate` calls doesn't cause warnings :)

Reply:
okay, makes sense, I will remove "eos" from generate calls to avoid deprecation warning 😄